ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/metrics/MetricsCollector.scala

```diff
@@ -29,10 +29,11 @@ object MetricsCollector {
       metrics: List[Metric[T]],
       logInterval: FiniteDuration,
       reporter: MetricReporter,
+      damlMetrics: Option[ExposedMetrics[T]] = None,
```

Comment:
I am not sure that qualifying the dropwizard metrics by the word "daml" helps make it clearer.

Reply:
Potential alternative names:
- exposedMetrics
- graphedMetrics
- dropwizardMetrics

Reply:
You're right - I forgot to change the name when renaming the class itself from `DamlMetrics` to `ExposedMetrics`.
Fixed now.

Reply:
There are couple more occurrences of that though.

Reply:
Ahh right. Sorry, I'm on it!

Reply:
Fixed.